pkg/verify/sct.go

```diff
@@ -29,16 +30,21 @@ import (
 // leaf certificate, will extract SCTs from the leaf certificate and verify the
 // timestamps using the TrustedMaterial's FulcioCertificateAuthorities() and
 // CTLogs()
-func VerifySignedCertificateTimestamp(leafCert *x509.Certificate, threshold int, trustedMaterial root.TrustedMaterial) error { // nolint: revive
+func VerifySignedCertificateTimestamp(chains [][]*x509.Certificate, threshold int, trustedMaterial root.TrustedMaterial) error { // nolint: revive
+	if len(chains) == 0 && len(chains[0]) == 0 && chains[0][0] == nil {
```

Comment:
```suggestion
	if len(chains) == 0 || len(chains[0]) == 0 || chains[0][0] == nil {
```

`len(chains)` has to be `>= 1` for us to check `chains[0]`?

Reply:
Nice catch!